### PR TITLE
Add frontend for Delete and Edit buttons

### PR DESCRIPTION
**Pull request by Froggerson**
image
image
Functionality with the backend has not been added yet.

### CODE REVIEW

File: frontend/src/flashcards/Flashcard.js [hunk 1]
```diff
@@ -1,56 +1,84 @@
...
+    this.handleClick = this.handleClick.bind(this);
+    this.state = { isFlipped: false };
+    this.handleDeleteCard = this.handleDeleteCard.bind(this);
+    this.handleEditCard = this.handleEditCard.bind(this);
+  }
+
+  /**
+   * handleClick allows the flashcard to be flipped over
+   * upon being clicked on.
+   */
```
Comment (ESaracay) [inline frontend/src/flashcards/Flashcard.js:42]: /** => /*

File: frontend/src/flashcards/Flashcard.js [hunk 2]
```diff
@@ -1,56 +1,84 @@
...
-			</div>
-		);
-	}
+  render() {
+    return (
+      <div
+        className="Flashcard"
+        onClick={this.handleClick}
+        style={{ height: "350px", width: "300px", margin: "15px" }}
+      >
```
Comment (ESaracay) [inline frontend/src/flashcards/Flashcard.js]: The style guide for html says that you shouldn't have a > alone on a line. It should go with the line before.

File: frontend/src/flashcards/Folder.css [hunk 3]
```diff
@@ -15,8 +15,18 @@
...
 }
-.Folder {
+.folder {
   width: 300px;
   height: 350px;
   margin: 20px;
+  display: flex;
+  flex-direction: column;
+}
+.icons {
```
Comment (ESaracay) [inline frontend/src/flashcards/Folder.css]: There should be a space between rules.

File: frontend/src/flashcards/Flashcard.js [hunk 4]
```diff
@@ -1,56 +1,84 @@
...
+
+  /**
+   * handleClick allows the flashcard to be flipped over
+   * upon being clicked on.
+   */
+  handleClick(e) {
+    this.setState((prevState) => ({ isFlipped: !prevState.isFlipped }));
+  }
+
+  handleDeleteCard(e) {}
```
Comment (zmr) [inline frontend/src/flashcards/Flashcard.js]: Are these handlers still to be implemented? If so, please add a //TODO comment here

File: frontend/src/flashcards/Flashcard.js (hunk 2 above)
Comment (tylerhou) [reply frontend/src/flashcards/Flashcard.js]: This is how prettier formats JSX, so it's fine.

**Pull request merged**